### PR TITLE
Fixes imports of common packages. Cleans up unused common functions.

### PR DESCRIPTION
**Pull request by joeldudley**
* Instead of `common2`/`common3`/etc., uses `obscurocommon` and `nodecommon` (`common` is taken by geth's `common`)
* Deletes some unused common functions
* Moves host code into host package (was previously directly under `obscuronode` directory)

### CODE REVIEW

**Comment by tudor-malene** [comment]
@joeldudley , this is purely a refactor PR?
I wonder if we can find better names, such that we don't clash on "common" any more.

**Comment by joeldudley** [comment]
@tudor-malene Yeah, ok, good idea. We can't control the name of Geth's `common` package, so I'll use `obscurocommon` and `nodecommon` for our packages.

File: integration/ethereummock/utils.go [hunk 1]
```diff
@@ -2,12 +2,12 @@ package ethereummock
...
 import (
 	"github.com/ethereum/go-ethereum/core/types"
-	common2 "github.com/obscuronet/obscuro-playground/go/common"
+	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
 )
 
-// LCA - returns the least common ancestor of the 2 blocks
-func LCA(blockA *types.Block, blockB *types.Block, resolver common2.BlockResolver) *types.Block {
-	if resolver.HeightBlock(blockA) == common2.L1GenesisHeight || resolver.HeightBlock(blockB) == common2.L1GenesisHeight {
+// LCA - returns the least nodecommon ancestor of the 2 blocks
```
Comment (otherview) [inline integration/ethereummock/utils.go:8]: nit: common

**Review by otherview — APPROVED** [review]
LGTM.

There is a tiny nit like GenesisHash being moved to the enclave package, but I don't think it's a blocker to merge.

One bit I would reference is the use of path for package names "obscurocommon" package name in "obscuro/common". We should probably package it in a different way. But I don't think we need to do it on this PR, and probably any refactor will be more evident as we add more things.

File: integration/ethereummock/utils.go (hunk 1 above)
Comment (joeldudley) [reply integration/ethereummock/utils.go]: Addressed.

**Comment by joeldudley** [comment]
@otherview I deleted the new `GenesisHash`. Agree the package names could perhaps be better, but will be easy to refactor later on now.

**Pull request merged**